templates/repo/commit_status.tmpl

```diff
@@ -1,4 +1,5 @@
 <!-- make sure this matches the color logic in web_src/js/components/DashboardRepoList.vue -->
+<span class="gt-df gt-ac">
```

Comment:
It's not right to put `gt-df gt-ac` everywhere. Only the parent container needs the "align-items: center"

Reply:
Yes, also this `gt-df gt-ac` is essentially a noop because it wraps the child exactly.